### ISSUE TITLE
[3.4.2] Errors on Installation

### ISSUE THREAD

**Issue by hostianer** [contao/core#7598]
MAMP Pro PHP 5.6.2, local on OSX.
Also tested with PHP 5.5.18

Version 3.4.0 is working without problems.

If i go to the /install.php i`m getting these errors:
```
Notice: Undefined index: HTTP_X_FORWARDED_HOST in /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php on line 302 

Notice: Undefined index: SSL_SESSION_ID in /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php on line 313 

Notice: Undefined index: HTTPS in /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php on line 313 

Notice: Undefined index: HTTPS in /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php on line 313 

Warning: Cannot modify header information - headers already sent by (output started at /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php:302) in system/modules/core/library/Contao/Controller.php on line 2125 
#0 [internal function]: __error(2, 'Cannot modify h...', '/Applications/M...', 2125, Array) 
#1 system/modules/core/library/Contao/Controller.php(2125): header('HTTP/1.1 303 Se...') 
#2 system/modules/core/controllers/BackendInstall.php(315): Contao\Controller::reload() 
#3 system/modules/core/controllers/BackendInstall.php(80): Contao\BackendInstall->acceptLicense() 
#4 contao/install.php(24): Contao\BackendInstall->run() 
#5 {main} 

Warning: Cannot modify header information - headers already sent by (output started at /Applications/MAMP/htdocs/pixelianer/system/modules/core/library/Contao/Environment.php:302) in system/modules/core/library/Contao/Controller.php on line 2126 
#0 [internal function]: __error(2, 'Cannot modify h...', '/Applications/M...', 2126, Array) 
#1 system/modules/core/library/Contao/Controller.php(2126): header('Location: http:...') 
#2 system/modules/core/controllers/BackendInstall.php(315): Contao\Controller::reload() 
#3 system/modules/core/controllers/BackendInstall.php(80): Contao\BackendInstall->acceptLicense() 
#4 contao/install.php(24): Contao\BackendInstall->run() 
#5 {main}  
```

**Comment by xchs** [comment]
This has already been fixed (see #7593).

**Comment by hostianer** [comment]
You`re refereing to my other issue, i had the wrong link from github.

But as you can see the error seems not to be fixed for 3.4.2

**Comment by Toflar** [comment]
> But as you can see the error seems not to be fixed for 3.4.2

Why should it? It's assigned to milestone 3.4.3...

**Comment by hostianer** [comment]
Ahhh okidoke, thank you :+1:
